CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 ## [Unreleased]
 
 ## [1.6.1] - 2017-08-24
+### Added
```

Comment:
This should be added up underneath unreleased. We do not add features to already released versions.

Reply:
Yeah, I don't know how those got there. Thanks!